### PR TITLE
Add Pulp 2.15 config

### PR DESCRIPTION
**Pull request by ehelms**
I am looking to use tool_belt for handling some aspects of Pulp releases. At this time it is unclear what functionality will be used so starting with the basic environment setup for repositories.

### CODE REVIEW

**Comment by akofink** [comment]
:rofl: https://community.theforeman.org/t/friendly-reminder-set-pull-request-descriptions/8550 :see_no_evil:

File: configs/pulp_2.15.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+:project: pulp
+:github_org: pulp
+:releases:
```
Comment (akofink) [inline configs/pulp_2.15.yaml]: Prior releases? Just basing this off my Katello knowledge.

File: configs/pulp_2.15.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  :2.15.3:
+    :redmine_release_id: 59
```
Comment (akofink) [inline configs/pulp_2.15.yaml]: code_name ?

File: configs/pulp_2.15.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  :pulp_python:
+    :main: true
+    :branch: 2.0-release
+    :repo: https://github.com/pulp/pulp_python
+    :version_branch: true
+  :pulp_ostree:
+    :main: true
+    :branch: 1.3-release
+    :repo: https://github.com/pulp/pulp_ostree
+    :version_branch: true
```
Comment (akofink) [inline configs/pulp_2.15.yaml]: Koji tags?

**Comment by ehelms** [comment]
Are you pointing me to that because you don't find the commit message clear or just because this literally does not set a description on the pull request first comment? I just want to understand so I can make things clearer.

**Comment by akofink** [comment]
Also, pulp_nightly?

**Comment by akofink** [comment]
It literally has no description. I'm pretty sure that's what @lzap was talking about.

**Comment by ehelms** [comment]
I agree that descriptions are useful when the commit message is unclear. But the description to begin with is "Add Pulp 2.15 config". That is why I ask if that is unclear, so that I can expand now and with forethought for the future.

Also setting this to WIP since this is net-new for Pulp and may not even go in.

**Comment by akofink** [comment]
Yeah, the commit subject is pretty straightforward, but it says nothing about the motivation for the change.

**Comment by pcreech** [comment]
As far as the pulp related config items, repos and branches look good for the 2.15 line

**Pull request merged**